src/components/Article.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function Article({ title, date, text }) {
+  return (
+    <article>
+      <h1>{title}</h1>
+      <span>{date}</span>
+      <p>{text}</p>
+    </article>
+  );
+}
```

Comment:
arrow function would save few lines of code with implicit return:
```suggestion
export const Article = ({ title, date, text }) => (
  <article>
    <h1>{title}</h1>
    <span>{date}</span>
    <p>{text}</p>
  </article>
);
```